include/mbedtls/ecjpake.h

```diff
@@ -71,6 +71,7 @@ typedef struct
     const mbedtls_md_info_t *md_info;   /**< Hash to use                    */
     mbedtls_ecp_group grp;              /**< Elliptic curve                 */
     mbedtls_ecjpake_role role;          /**< Are we client or server?       */
+    int point_format;                   /**< Format for point export        */
```

Comment:
This should be set to `MBEDTLS_ECP_PF_UNCOMPRESSED` by default in `mbedtls_ecjpake_init()`. I didn't check but I think this is why the tests are failing (currently this is no initialized at all).